dynamic-tools/zap-extensions/wso2-jira-issue-creator/wso2jiraissuecreator/CredentialForm.java

```diff
@@ -0,0 +1,160 @@
+
+package org.zaproxy.zap.extension.wso2jiraissuecreator;
+
+import org.parosproxy.paros.Constant;
+
+import javax.swing.*;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.util.Properties;
+
+public class CredentialForm extends JFrame {
+
+    /**
+     * Creates new form CredentialForm
+     */
+    public CredentialForm() {
+        initComponents();
+    }
+
+    /**
+     * This method is called from within the constructor to initialize the form.
+     * WARNING: Do NOT modify this code. The content of this method is always
+     * regenerated by the Form Editor.
+     */
+    @SuppressWarnings("unchecked")
+    // <editor-fold defaultstate="collapsed" desc="Generated Code">//GEN-BEGIN:initComponents
+    private void initComponents() {
+
+        jLabel1 = new JLabel();
+        tbUrl = new JTextField();
+        tbUsername = new JTextField();
+        jLabel2 = new JLabel();
+        jLabel3 = new JLabel();
+        tbPassword = new JPasswordField();
+        btnAdd = new JButton();
+
+        setDefaultCloseOperation(WindowConstants.HIDE_ON_CLOSE);
+
+        jLabel1.setText("Jira URL");
```

Comment:
These label values should be put in a property file in Resources and then read from here